### DISCUSSION TITLE
Sponsorship one-time

### DISCUSSION THREAD

**Discussion by javierdev-pro18** [petyosi/react-virtuoso#841] [General]
@petyosi Hey hope you are good,  I already pay a sponsorship want to know how we can set-up a session and pair in the problem, I can not add a sandbox.

**Comment by petyosi** [comment]
Thanks for the sponsorship, 

The most effective way for me to help you is to isolate your problem in a codesandbox - pick one of the examples from the docs, open it in a sandbox by pressing the cube at the bottom right, and adjust it accordingly. Save the sandbox and send it to me. You can contact me through the email in my [github profile](https://github.com/petyosi/) to discuss it further.

Notice that, most likely, you don't need Apollo to reproduce the problem. You can simulate async data with `setTimeout`. 

I think we should be able to solve this over email. Pairing without a prior understanding on my side can turn into a waste of your time, something that I don't want to cause.
